src/devtools.js

```diff
@@ -27,5 +25,6 @@ export function plugins(options) {
       new webpack.NamedModulesPlugin(),
       new webpack.HotModuleReplacementPlugin(),
       new webpack.NoEmitOnErrorsPlugin(),
+      new webpack.LoaderOptionsPlugin({options: {}}),
```

Comment:
Maybe add comment why this is necessary? I also wonder... is it because the eslint plugin (at least that's why I needed to add it)?